bedrock/products/views.py

```diff
@@ -64,13 +81,18 @@ def vpn_landing_page(request):
 
 @require_safe
 def vpn_pricing_page(request):
-    template_name = "products/vpn/pricing.html"
+    template_name = "products/vpn/pricing-refresh.html" if request.locale == "en-US" else "products/vpn/pricing.html"
```

Comment:
@craigcook open question: how should we handle launching the refreshed dedicated pricing page? We're no longer linking to it from the main landing page, but if we send any folks to it directly, they'll get the redesigned page if they have en-US set as a locale.

Maybe this is OK? Or might they want to test it separately? I dunno the answer, but maybe we should ask someone. Thoughts?

Reply:
hmm, good question... the original plan was to show it to all en-US anyway so maybe that's still fine? Let's check with Dan.

Reply:
Confirmed: we're OK with all en-US traffic getting the redesigned pricing page